### ISSUE TITLE
Working Repo?

### ISSUE THREAD

**Issue by jesouhaite08** [musicode/react-native-pure-chat-view#1]
Is this working? Are there any docs on how to get this up and running?

**Comment by musicode** [comment]
yes, it like the screenshoot, but the repo is only used by me for now.

![image](https://user-images.githubusercontent.com/2732303/55534042-c7bc0b80-56e5-11e9-9954-35ace72ce0ae.png)

**Comment by jesouhaite08** [comment]
I looked at your example and code and don't see a place to connect it to my account. Can you please point me to where this happens?

**Comment by Romkin7** [comment]
Hi, thak you jesouhaite08 for your question. i'm also intrested in how to implement this code in my react app. so far been using this just in html. :)

**Comment by musicode** [comment]
@jesouhaite08  @Romkin7  do you like the user interface?

**Comment by musicode** [comment]
my english is not very good, so it's difficult to write a document to show how to use the repo.

the usage is not very simple, you have to configurate your image loader for ios and android.
